src/main/java/org/reaktivity/reaktor/Reaktor.java

```diff
@@ -110,6 +111,11 @@ public void detach(
         }
     }
 
+    public void visit(Consumer<Controller> action)
+    {
+        this.controllersByKind.values().forEach(action);
+    }
+
```

Comment:
I use it to bind the controllers by name in ry.java: https://github.com/reaktivity/ry.java/pull/1/files#diff-a2019a4e55b641a4f7c005781a3086c1.

Thoughts?